content/installation/net/install/NetInstallation.md

```diff
@@ -6,58 +6,56 @@ tags: "installation agent .NET"
+Many users are curious about the changes made by the .NET agent and the impacts these changes may have. In all respects, the Contrast .NET agent installer is a normal Windows application installer built using standard MSI technology. The .NET agent installer validates that the target server satisfies several requirements (e.g., the operating system is Windows Server 2008 R2 or greater). If all requirements are met, the installer registers the .NET agent as a standard Windows program and makes the following changes.
 
-- Places the agent’s files on a disk in the specified install location (e.g., *C:\ProgramFiles\Contrast\dotnet*). This includes several dynamic link library (DLLs) and executables, such as the background Windows service that drives agent behavior. 
-- Creates the specified data directory for the agent that is primarily used to store agent log files (e.g., *C:\ProgramData\Contrast\dotnet*). 
+- Places the agent’s files on a disk in the specified install location (e.g., *C:\Program Files\Contrast\dotnet*). This includes several dynamic link libraries (DLLs) and executables, such as the background Windows service that drives agent behavior. 
+- Creates the specified data directory for the agent that's primarily used to store agent log files and configuration (e.g., *C:\ProgramData\Contrast\dotnet*). 
 - Registers the agent’s background Window service with the operating system.
 - Reads the *DotnetAgentSettings.ini* file to customize the agent’s configuration file with details necessary to communicate with the Contrast application (e.g., API key).
 - Registers several agent assemblies with the .NET global assembly cache so they can be loaded by instrumented applications.
 - Starts the agent’s background Windows service and Tray (UI) application. This service has a number of responsibilities: 
-  - Preparing the environment for instrumentation by registering the agent’s profiler component with IIS through environment variables and restarting IIS. This causes the CLR to load the agent’s profiler, which is responsible for instrumenting analyzed applications. 
-  - Communication with the Contrast interface.
-  - Communication with Profiler and Sensor components through local named pipes. 
+  - Preparing the environment for instrumentation by registering the agent’s profiler component with IIS through environment variables, and restarting IIS. This causes the CLR to load the agent’s profiler, which is responsible for instrumenting analyzed applications. 
+  - Communication with the Contrast UI.
+  - Communication with profiler and sensor components through local named pipes. 
```

Comment:
It looks like we inconsistently capitalize "profiler" in other articles. I've put both terms in lowercase here.